src/img/crest.svg

```diff
@@ -1,514 +1,4 @@
-	c0-0.1,0.2-0.2,0.3-0.1c0,0,0.1,0,0.1,0c0,0,0.1,0,0.2,0.1c0.1,0.1,0.1,0.1,0.2,0.2c0.2,0.1,0.4-0.1,0.6-0.2
-	c0.2-0.2,0.4-0.5,0.7-0.7c-0.6-0.3-0.9-0.9-1-1.1c0-0.1-0.1-0.2-0.1-0.2c0,0,0,0.1,0,0.1c-0.1,0-0.1-0.1-0.1-0.2c0,0,0,0,0,0
-	c0,0-0.1,0-0.1,0.1c0,0,0-0.2,0-0.3c-0.3,0.1-0.6-0.1-0.4-0.4c0,0.2,0.3,0.3,0.4,0.2c-0.2,0-0.2-0.2-0.1-0.3c0,0,0,0,0,0
-	c0,0,0,0-0.1,0c0,0,0,0-0.1,0c-0.1,0.1-0.1,0.1-0.1,0c0,0,0,0-0.1-0.1c0,0,0,0,0,0c-0.1,0,0,0.1,0,0.1c-0.2-0.1-0.1-0.2-0.1-0.3
-	c0,0,0-0.1,0-0.1c0-0.1,0.1-0.2,0.3-0.1c0,0,0,0,0,0c0,0,0,0,0-0.1c0,0,0,0,0,0c-0.1,0-0.2-0.1-0.2-0.2c0,0,0-0.1,0-0.1
-	c0,0-0.1,0-0.1-0.1c0.2-0.2,0.3,0,0.3,0.1c0,0,0,0,0,0c0,0,0.1,0,0.1,0c0.1,0,0.2,0.2,0.2,0.3c0,0,0,0,0,0c0,0,0-0.1,0-0.1
-	c0,0,0,0,0,0c-0.1-0.1,0-0.1,0-0.2c0,0,0-0.1,0.1-0.1c0-0.1-0.1-0.1-0.1-0.2c0.1,0,0.2,0.1,0.3,0.2c0,0,0,0.1,0,0.2
-	c0,0.2-0.1,0.3-0.1,0.5c0,0.2,0,0.3,0.2,0.4c0.2,0.1,0.3,0.2,0.4,0.4c0.2,0.2,0.4,0.4,0.4,0.5c0.1,0.1,0.2,0.2,0.3,0.3
-	c0.1-0.6,0.7-0.6,0.7-1.1c0-0.2-0.4-0.2-0.6-0.1c-0.2,0.1-0.4,0-0.4-0.1c0-0.1,0-0.2,0.1-0.2c0.2,0,0.2,0,0.4,0
-	c0.2,0,0.1-0.1,0.1-0.2c0-0.2-0.3-0.2-0.4-0.1l-0.1,0.2l-0.2-0.2c0,0,0,0-0.1,0c-0.1-0.1-0.1-0.2-0.1-0.3c0-0.1,0-0.3,0.2-0.2
-	c0.2,0,0.2,0,0.4-0.1c0.1-0.1,0.1-0.1,0.3-0.1C61.5,40.2,61.6,40.1,61.7,40z M58.7,37.9c0.1,0.1,0.1,0.3,0.1,0.5
-	c-0.2,0-0.3-0.1-0.4-0.2v4c0.2,0,0.4,0.1,0.4,0.2c0,0.1,0,0.3-0.2,0.5c0.2,0,0.4,0.1,0.6,0.3c-0.2,0.2-0.4,0.3-0.6,0.3
-	c0.2,0.2,0.2,0.3,0.2,0.5c0,0.1-0.2,0.2-0.4,0.2v4.7c0.1-0.1,0.3-0.2,0.5-0.2c0,0.2-0.1,0.4-0.2,0.5h2.9c0-0.2,0.1-0.3,0.2-0.4
-	c0.1-0.1,0.3,0,0.5,0.2c0-0.2,0.1-0.4,0.3-0.6c0.2,0.2,0.3,0.4,0.3,0.6c0.2-0.2,0.3-0.2,0.5-0.2c0.1,0,0.2,0.2,0.2,0.4h3.4
-	c-0.1-0.1-0.2-0.3-0.2-0.5c0.2,0,0.3,0,0.5,0.1c-0.3-2-1-3.9-2-5.6c-0.1,0-0.2,0-0.3,0c-0.1-0.1-0.2-0.2-0.2-0.5
-	c-0.2,0.1-0.4,0.2-0.6,0.1c0-0.3,0.2-0.4,0.3-0.6c-0.2,0-0.4-0.1-0.4-0.3c0-0.1,0-0.2,0.1-0.3C62.6,39.9,60.7,38.6,58.7,37.9z
-	 M58,37.4l0.2,0.1c1.2,0.3,2.3,0.8,3.3,1.5c1.1,0.7,2,1.6,2.9,2.7c1.7,2.2,2.8,4.9,3.1,7.7l0,0.2H58V37.4z M57.8,37.2v12.6h10
-	c-0.2-3-1.4-5.9-3.2-8.2C62.8,39.2,60.4,37.7,57.8,37.2z M58.2,36.6c0.2-0.1,0.5,0.1,0.6,0.5c2.3,0.7,4.4,2.2,6.1,4.3
-	c0.1,0.1,0.2,0.2,0.3,0.4c0.2-0.1,0.4-0.2,0.7-0.2c-0.1,0.3-0.2,0.5-0.4,0.6c1.4,2,2.3,4.3,2.6,6.7c0.4,0,0.5,0.4,0.4,0.6
-	c-0.1,0.1-0.2,0.2-0.3,0.2c0,0.1,0,0.1,0,0.1c0.1,0.1,0.1,0.3,0.1,0.5c-0.2,0-0.4-0.1-0.5-0.2h-0.1c0,0.2-0.1,0.3-0.2,0.4
-	c-0.2,0.1-0.5,0-0.6-0.4h-4c0,0.2-0.1,0.4-0.3,0.6c-0.2-0.2-0.3-0.4-0.3-0.6h-3.5c-0.1,0.3-0.4,0.4-0.6,0.3
-	c-0.1-0.1-0.2-0.2-0.2-0.3h-0.1c-0.1,0.1-0.3,0.1-0.5,0.1c0-0.2,0.1-0.4,0.1-0.5v-0.1c-0.2,0-0.3-0.1-0.4-0.2
-	c-0.1-0.2,0-0.5,0.4-0.6v-5.3c-0.2,0-0.4-0.1-0.6-0.3c0.2-0.2,0.4-0.3,0.6-0.3v-4.9c-0.3-0.1-0.4-0.4-0.3-0.6
-	c0.1-0.1,0.2-0.2,0.3-0.2v0c-0.1-0.2-0.2-0.4-0.2-0.6c0.2,0,0.4,0.1,0.5,0.2c0,0,0.1,0,0.1,0C58,36.7,58.1,36.6,58.2,36.6z"/>
+<svg xmlns="http://www.w3.org/2000/svg" width="115px" height="96px" viewBox="0 0 115 96" focusable="false">
```

Comment:
Can these svg files now be removed because the code is now in the icons macro

Reply:
We can remove them @rmccar

Reply:
Ok yeah fair enough